hrui/templates/CRM/common/footer.tpl

```diff
@@ -28,17 +28,29 @@
   {if !empty($contactId)}
     {include file="CRM/common/contactFooter.tpl"}
   {/if}
-
   <div class="crm-footer" id="civicrm-footer">
     {* PCHR-1323 - Display CiviHR version info. *}
     {ts}Powered by CiviHR {/ts} {civihrVersion}.
 
+    {php}
```

Comment:
Can you change this to use something like [this](https://github.com/civicrm/civihr/blob/cc7514b47b69f3c1f86915d46e73ced3b4dc3b60/hrjobcontract/CRM/Hrjobcontract/Page/View/Summary.tpl#L52). It's better to not call Drupal functions directly in CiviHR since we're still trying to be CMS-agnostic